backend/README.md

```diff
@@ -117,6 +117,41 @@ returns
 ```
 **Note:** If the user ID specified is that of the logged in user, the full user info will be given
 
+### Start the Payment flow to add funds to user account
```

Comment:
Looks good, should there be a way for a user to transfer funds from Rafflebay back to their stripe account?